Dockerfile

```diff
@@ -64,6 +81,20 @@ RUN echo "Cloning njs $NJS_VERSION" \
   && ./configure \
   && make
 
+RUN echo "Cloning nginx-jwt-module $NGINX_JWT_VERSION" \
+  && git clone --depth 1 --branch $NGINX_JWT_VERSION https://github.com/max-lt/nginx-jwt-module.git
+
+RUN echo "Installing libjwt $LIBJWT_VERSION" \
```

Comment:
oof libjwt is a compile from source? Have we checked how long this takes? I do see though that 1.15.3 doesn't appear to be available on Debian stable, so maybe this is just what we have to do.